sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+    {
+        "buffer": "chopin-left",
+        "speed": 1.0,
+        "loop": 0.0,
+
+        "synthDef": {
+            "ugen": "flock.ugen.out",
+            "bus": 16,
+            "expand": 1,
+            "sources": {
+                "ugen": "flock.ugen.filter.biquad.lp",
+                "freq": 1000,
+                "source": {
+                    "id": "volume-tracker",
+                    "ugen": "flock.ugen.amplitude",
+                    "source": {
+                        "ugen": "flock.ugen.playBuffer",
+                        "loop": "{that}.options.loop",
+                        "speed": "{that}.options.speed",
+                        "buffer": "{that}.options.buffer"
+                    }
+                }
+            }
+        }
+    }
+Example x: _An example of how signal processing programs are represented in Flocking. This program's signal is used to determine the overall compositional structure of_ Font Màgica. _It represents an instance of "declarative" data flow programming, where a collection of unit generators are specified within a loosely schematized data structure to: 1) read an audio file's signal from a "bus", 2) track the volume of the incoming audio, and 3) smooth the output of the amplitude tracker using a low-pass filter._
+
+![Image of the signal-processing "synth" used to determine the overall compositional structure of Font Màgica.](../images/raindrop-macro-structure-synth-diagram.png)
+
+_Figure x: An illustration of the meta-level structure of_ Font Màgica _generated using Flocking's interactive textual/graphical programming environment._
+
+Secondly, Flocking and Aconite programs are constructed using a rich network of "landmarks," or names, which provide stable references into every point of composition in a program. Traditionally, names in software are either inward-facing constructs or pseudo-legal contracts, designed to make it easier for a team of programmers to understand their code or to provide other programmers with a fixed repertoire of actions that can be performed with a piece of software. Computation fastidiously covers its tracks; composition too often simply erases its components. In contrast, works made with Flocking and Aconite bind each point of composition and behaviour to a name, a stable referent to which changes can be targeted, items can be removed, or new activities injected. Where ordinary function composition is smooth, leaving no traces, textures, or seams, my programs are woven with names; they provide the means to open the seams, unravel the material, and pick up the computational stitches to make something new.
```

Comment:
Perhaps, continuing the _reknitting_ line of thought & our Gchat discussion of decompilation, computation typically conceives itself as a suit of armor rather than a sweater; it expects the use of specialized expert tools and techniques (decompilers, reverse engineering) rather than everyday ones (text editors, `view source`). :hammer: :hammer: :hammer: